package.json

```diff
@@ -22,12 +22,18 @@
   "main": "gs.js",
   "scripts": {
     "test": "npm run lint && node test/test.all",
-    "lint": "eslint ."
+    "lint": "eslint .",
+    "build-browser": "webpack --progress --colors --mode=none"
   },
   "engines": {
     "node": ">=6.0.0"
   },
+  "browser": {
+    "./lib/fs.provider.js": false,
+    "./node_modules/mkdirp": false
+  },
   "dependencies": {
+    "localStorage": "^1.0.3",
```

Comment:
We need no this dependency, browser have built-in `window.localStorage`, at backend we will never use this provider

Reply:
You're right. I've moved it dev-dependencies to use it only for testing